distribution/lib/Standard/Database/0.0.0-dev/src/Internal/DB_Data_Link_Helpers.enso

```diff
@@ -12,3 +17,56 @@ data_link_connection_parameters (source : Data_Link_Source_Metadata) -> Connecti
     audit_mode = if Enso_User.is_logged_in then "cloud" else "local"
     options_vector = [["enso.internal.audit", audit_mode]] + (if related_asset_id.is_nothing then [] else [["enso.internal.relatedAssetId", related_asset_id]])
     Connection_Options.Value options_vector
+
+## PRIVATE
+save_table_as_data_link table destination on_existing_file:Existing_File_Behavior =
+    data_link_setup = table.connection.data_link_setup.if_nothing <|
+        Error.throw (Illegal_Argument.Error "Saving a Table as data link is currently not supported in this backend.")
+
+    # For a trivial query we return the table name.
+    link_type = if table.is_trivial_query fail_if_not_found=False then ..Table table.name else
```

Comment:
```suggestion
    link_type = if table.is_trivial_query fail_if_not_found=False then ..Table table.name else
```

We shouldnt use autoscoping here - we should have concrete type.

Reply:
I thought it makes the code a bit more terse. But ok, I can change it, I guess maybe it obscures it a bit too much indeed.